sixtyfps_compiler/expression_tree.rs

```diff
@@ -43,7 +43,7 @@ pub enum BuiltinFunction {
     ColorBrighter,
     ColorDarker,
     Rgb,
-    ImplicitItemSize,
+    ImplicitLayoutInfo,
```

Comment:
With ImplicitSize I can see the counter part being the "set" size. With the LayoutInfo, is there a counter-part as well?

AFAICS the implementation basically calls `layouting_info()` on the item vtable, so perhaps `ItemLayoutInfo` is cleaner.

Anyway, just a nitpick - feel free to ignore this one :)

Reply:
yes, when `maximum-width` or `minimum-height` or so are set, that's the explicit.